docs/static/streamer/streamer.ts

```diff
@@ -366,9 +366,7 @@ function onYouTubeIframeAPIReady() {
 
     function loadToolbox() {
         const config = readConfig();
-        // tslint:disable-next-line: no-inner-html
         toolbox.innerHTML = "";
-        // tslint:disable-next-line: no-inner-html
         paintbox.innerHTML = "";
```

Comment:
Note that the new no-inner-html rule allows assigning the empty string since that is always safe.